packages/foreman/rubygem-rbvmomi2/rubygem-rbvmomi2.spec

```diff
@@ -3,14 +3,16 @@
 
 Name: rubygem-%{gem_name}
 Version: 3.6.0
-Release: 2%{?dist}
+Release: 3%{?dist}
 Summary: Ruby interface to the VMware vSphere API
 License: MIT
 URL: https://github.com/ManageIQ/rbvmomi2
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
 # start specfile generated dependencies
 Requires: ruby >= 2.4.1
+Requires: rubygem(optimist) >= 3.0
+Requires: rubygem(optimist) < 4.0
```

Comment:
Please put this outside the "specfile generated dependencies" block, as otherwise it will be removed on the next automatic bump of the spec.

Reply:
Ack, done